### PR TITLE
Migrate userChip tests to react-testing-library

### PR DESCRIPTION
**Pull request by LukasLohoff**
### Description:

Migrates `UserChip` tests.

@terrestris/devs Please review

### CODE REVIEW

File: src/UserChip/UserChip.spec.tsx [hunk 1]
```diff
@@ -1,57 +1,62 @@
 import testImage from  '../../assets/user.png';
-import TestUtil from '../Util/TestUtil';
 
 import UserChip from './UserChip';
+import { render, screen } from '@testing-library/react';
+import * as React from 'react';
+import '@testing-library/jest-dom';
```
Comment (simonseyock) [inline src/UserChip/UserChip.spec.tsx:6]:
```suggestion
```

File: src/UserChip/UserChip.spec.tsx [hunk 2]
```diff
@@ -1,57 +1,62 @@
...
+    const { container } = render(<UserChip />);
+    expect(container).toBeVisible();
   });
 
   it('determines initials from given user name', () => {
-    wrapper.setProps({userName: 'Shinji Kagawa'});
-    const test = wrapper.instance().getInitials();
-    expect(test).toBe('SK');
+    render(<UserChip userName="Shinji Kagawa" />);
+    const chip = screen.getByText('SK').parentElement;
```
Comment (simonseyock) [inline src/UserChip/UserChip.spec.tsx:22]:
```suggestion
    const chip = screen.getByText('SK');
```
wouldn't that be enough?

**Review by simonseyock — COMMENTED** [review]
Looks great, only 2 minor remarks.

File: src/UserChip/UserChip.spec.tsx (hunk 2 above)
Comment (LukasLohoff) [reply src/UserChip/UserChip.spec.tsx:21]: in this case, yes

**Comment by simonseyock** [comment]
The new eslint plugin also has some remarks :dizzy:

**Comment by LukasLohoff** [comment]
> The new eslint plugin also has some remarks

Fixed that. The rule `prefer-in-document` is kind of annoying though. Should be a warning IMHO, not an error.

**Comment by LukasLohoff** [comment]
Thanks for the review. Merging this now.

**Pull request merged**